internal/server/data/encryption_key.go

```diff
@@ -26,7 +31,12 @@ func (e *encryptionKeysTable) ScanFields() []any {
 	return []any{&e.Algorithm, &e.CreatedAt, &e.DeletedAt, &e.Encrypted, &e.ID, &e.KeyID, &e.Name, &e.RootKeyID, &e.UpdatedAt}
 }
 
-func CreateEncryptionKey(tx WriteTxn, key *models.EncryptionKey) error {
+// StdlibTxn is a transaction that uses only the methods from the database/sql
+// package. It excludes the metadata that exists on the ReadTxn and WriteTxn.
+// StdlibTxn is for tables that are not scoped to an org.
+type StdlibTxn migrator.DB
```

Comment:
This is necessary because these functions are called by the migrator. We could remove this if we decided to only load the key after the migrations. If we want to allow migrations access to this key we need to do something like this.